connectors/plex.js

```diff
@@ -46,14 +46,27 @@ function setupLocalUI() {
  * Setup properties for UI available on https://app.plex.tv/web/app.
  */
 function setupRemoteUI() {
-	Connector.playerSelector = '[class^=MiniPlayerContainer-miniPlayer]';
+	Connector.playerSelector = '[class^=AudioVideoPlayerView-container]';
 
 	Connector.artistSelector = `${Connector.playerSelector} [class*=MetadataPosterTitle-title] > a:nth-child(1)`;
 
 	Connector.trackSelector = `${Connector.playerSelector} [class*=titlesContainer] > a`;
 
+	// for watch-it-later videos
+	Connector.artistTrackSelector = `${Connector.playerSelector} [class*=MetadataPosterTitle-title]`;
+
+	Connector.getTrack = () => {
+		if (Connector.getArtist()) {
+			return $(Connector.trackSelector).text();
+		}
+		Connector.filter = MetadataFilter.getYoutubeFilter();
```

Comment:
Is there a way to not create new filter object on every `getTrack` call?

Reply:
Can't this be done outside the getTrack scope?